src/lib/util/util.js

```diff
@@ -94,7 +95,27 @@ class Util {
 	}
 
 	/**
-	 * Applies an interface to a class|
+	 * Sets default properties on an object that aren't already specified.
+	 * @param {Object} def Default properties
+	 * @param {Object} given Object to assign defaults to
+	 * @returns {Object}
+	 * @private
+	 */
+	static mergeDefault(def, given) {
```

Comment:
Not really much reason that this code should be duplicated no? This is equivalent to the one Discord.js uses and it should probably be used instead of duplicating it specifically for Klasa, or at the very least:

```js
const { Util: { mergeDefault } } = require("discord.js"):

class Util {
  static mergeDefault (...args) {
    return mergeDefault(...args); // or Util.mergeDefault to dispute naming issues
  } 
}
```

Reply:
I tried that originally, but then it fails on this.mergeDefault in d.js...

Reply:
We talked about this on the server and determined It was probably best left alone to keep destructuring from breaking.